server/plugins/good-hoodie.js

```diff
@@ -0,0 +1,63 @@
+const Stream = require('stream')
+
+const log = require('npmlog')
+const qs = require('querystring')
+
+class HoodieTransform extends Stream.Transform {
+
+  constructor () {
+    super({ objectMode: true })
+  }
+
+  _transform (data, enc, next) {
+    if (data.event === 'error') {
+      log.error(
+        data.error.name,
+        new Date(data.timestamp).toISOString(),
+        data.error.message,
+        data.error
+      )
+    }
+
+    if (data.event === 'response') {
+      var path = data.path +
+        (Object.keys(data.query).length ? '?' + qs.stringify(data.query) : '')
+      log.http(
+        data.event,
+        new Date(data.timestamp).toISOString() + ' -',
+        data.source.remoteAddress + ' -',
+        data.method.toUpperCase(),
+        path,
+        data.statusCode,
+        data.responseTime + 'ms'
+      )
+    }
+
+    if (data.event === 'request' || data.event === 'log') {
+      var level = findLogLevel(Object.keys(log.levels), data.tags) || 'verbose'
+      log[level](
+        data.event,
+        new Date(data.timestamp).toISOString(),
+        data.tags.length ? data.tags : data.data,
+        data.tags.length ? data.data : ''
+      )
+    }
+
+    log.silly(data.event, data)
+    next(null)
```

Comment:
@mbad0la - one thing I just noticed -- just in case in the future we decide to chain more transforms after `HoodieTransform`, we should send `data` back with the callback, so this should be `next(null, data)`